src/utl/README.md

```diff
@@ -33,6 +33,13 @@ man
 
 ## Example scripts
 
+You may run various commands
+```
+man openroad; level 1
+man clock_tree_synthesis; level 2
+man CTS-0005; level 3
```

Comment:
The 'level #' are confusing - are they actually commands?

Reply:
Updated